### PR TITLE
[#930] Pluggable build mechanism

### PR DESCRIPTION
**Pull request by mcdoker18**
* Moved examples to the legion-examples repo
* Transform runner to a python package
* Integration is created using EDI server
* Used legion-sdk library

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    data_files=[('', ["README.md"])],
+    zip_safe=False,
+    entry_points={
+        'console_scripts': ['legion-mlflow-runner=mlflowrunner.runner:main'],
+    },
+    install_requires=[
+        # TODO: fix dependencies when we will publish the legion packages to pypi repo
+        # TODO: change branch to develop
+        # TODO: consider Pipenv usage
+        'legion-sdk @ git+https://github.com/legion-platform/legion.git@feat/930-pluggable-build-mechanism#egg=legion-sdk&subdirectory=legion/sdk',
```
Comment (kirillmakhonin) [inline setup.py]: OK for now

Comment (kirillmakhonin) [reply setup.py]: But pls change branch t exact tag before merge

File: helms/legion-mlflow/templates/tracking-server/mlflow-server.yaml [hunk 2]
```diff
@@ -47,7 +47,7 @@ spec:
 apiVersion: v1
 kind: Service
 metadata:
-  name: "{{ .Release.Name }}-legion-mlflow-tracking-server"
+  name: "mlflow"
```
Comment (kirillmakhonin) [inline helms/legion-mlflow/templates/tracking-server/mlflow-server.yaml:50]: Why did you remove release name?

**Review by kirillmakhonin — APPROVED** [review]

File: helms/legion-mlflow/templates/tracking-server/mlflow-server.yaml (hunk 2 above)
Comment (mcdoker18) [reply helms/legion-mlflow/templates/tracking-server/mlflow-server.yaml:50]: Will be fixed later

**Review by maxkochubey — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (mcdoker18) [reply setup.py]: Fixed
